### DISCUSSION TITLE
Removing embedded lyrics

### DISCUSSION THREAD

**Discussion by Virtuo1** [beetbox/beets#4687] [Q&A] [answered]
How do i remove embedded lyrics fetched with lyrics plugin? I have fallback: '' but im not sure if it is resetting it at all. How can i make sure that it is removing it?

**Comment by DjSlash** [comment, marked as answer]
Using fallback in that way is not meant for removing lyrics. If you want to remove lyrics, you can use `beet modify lyrics!` (see [modify](https://beets.readthedocs.io/en/stable/reference/cli.html#modify))
